.github/workflows/_reusable_helm.yml

```diff
@@ -0,0 +1,134 @@
+name: _Build and publish Helm Chart
+
+on:
+  workflow_call:
+    inputs:
+      release:
+        type: boolean
+        required: false
+        default: false
+      version:
+        type: string
+        required: true
+        default: "0.0.0-dev"
+
+jobs:
+  lint-chart:
+    runs-on: [self-hosted, shared]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.0
+      - name: Run chart-testing (lint)
+        run: ct lint --config .github/ct.yaml --check-version-increment=false
+
+  lint-docs:
+    runs-on: [self-hosted, shared]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run helm-docs
+        env:
+          HELM_DOCS_VERSION: "1.11.0"
+          HELM_DOCS_BASE_URL: "https://github.com/norwoodj/helm-docs/releases/download"
+        run: |
+          OS=$(uname)
+          # install helm-docs
+          curl -sSfL "${HELM_DOCS_BASE_URL}/v${HELM_DOCS_VERSION}/helm-docs_${HELM_DOCS_VERSION}_${OS}_x86_64.tar.gz" |
+          tar -xzf - helm-docs
+          # validate docs
+          ./helm-docs
+          git diff --exit-code
```

Comment:
I think those 2 should move to `pre-commit`, it would avoid waiting for CI to discover issues while it could have been discovered on commit